src/main/java/mil/dds/anet/resources/AttachmentResource.java

```diff
@@ -307,6 +308,10 @@ public static Map<String, Object> getAttachmentSettings() {
 
   @SuppressWarnings("unchecked")
   public static List<String> getAllowedMimeTypes() {
-    return (List<String>) getAttachmentSettings().get("mimeTypes");
+    // Get the allowed mime Types from dictionary
+    final List<Object> mimeTypes = (List<Object>) getAttachmentSettings().get("mimeTypes");
+    // Extract names
+    return mimeTypes.stream().map(element -> (String) ((HashMap<?, ?>) element).get("name"))
+        .toList();
```

Comment:
Code can be simplified:
```java
    // Get the allowed mime types from dictionary
    final var mimeTypes = (List<HashMap<String, ?>>) getAttachmentSettings().get("mimeTypes");
    // Extract names
    return mimeTypes.stream().map(element -> (String) element.get("name")).toList();
```

Reply:
Like in one liner? I thought it more readable like this.

Reply:
No, fewer class casts by a. declaring that you get a list of hashmaps, and b. using var to make the code DRY'er.

Reply:
Ah right, sorry did not realize you wrote the simplified version already in your comment.